platform/saivpp/vpplib/SwitchStateBase.h

```diff
@@ -992,7 +992,9 @@ namespace saivpp
 
         private:
             std::map<sai_object_id_t, std::list<sai_object_id_t>> m_acl_tbl_rules_map;
+            std::map<sai_object_id_t, std::list<std::string>> m_acl_tbl_hw_ports_map;
```

Comment:
There is already m_acl_tbl_grp_ports_map. If we see acl and tunterm_acl as two objects created from the same acl_table, it is possible to share the same mapping. even m_acl_swindex_map and m_tunterm_acl_swindex_map can be combined to one if we use std::pair or std::Array to keep both indexes in one

Reply:
Will consider merging some of these mappings as a future enhancement